support-frontend/assets/pages/supporter-plus-landing/twoStepPages/secondStepCheckout.tsx

```diff
@@ -93,6 +94,13 @@ export function SupporterPlusCheckout({
 		</Button>
 	);
 
+	const promotion = useContributionsSelector((state) =>
+		getPromotion(
```

Comment:
let's look at this in another PR - but I think `getPromotion` might actually be neater just taking the `state` value.